### PR TITLE
comment cleanup for kube-proxy

### PR DESCRIPTION
**Pull request by SataQiu**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
code cleanup for kube-proxy

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

**Comment by aojea** [comment]
nit/question, are the `On______Synced` descriptions accurate? should they be more descriptive??

```
 is called once all the initial event handlers were
// called and the state is fully propagated to local cache.
```

**Comment by SataQiu** [comment]
Thanks @aojea !
I refer to something like this https://github.com/kubernetes/kubernetes/blob/master/pkg/proxy/iptables/proxier.go#L580 、https://github.com/kubernetes/kubernetes/blob/master/pkg/proxy/iptables/proxier.go#L617
Do you have a better idea？

**Comment by aojea** [comment]
/lgtm
I didn't realize that those descriptions already exist
/assign @thockin @danwinship

File: pkg/proxy/userspace/proxier.go [hunk 1]
```diff
@@ -621,18 +621,26 @@ func (proxier *Proxier) serviceChange(previous, current *v1.Service, detail stri
...
 	proxier.serviceChange(oldService, service, "OnServiceUpdate")
 }
 
+// OnServiceDelete is called whenever deletion of an existing service
+// object is observed.
 func (proxier *Proxier) OnServiceDelete(service *v1.Service) {
 	proxier.serviceChange(service, nil, "OnServiceDelete")
 }
 
+// OnServiceSynced is called once all the initial even handlers were
```
Comment (danwinship) [inline pkg/proxy/userspace/proxier.go]: even**T** handlers

Probably needs to be fixed wherever you copied that from too...

Comment (SataQiu) [reply pkg/proxy/userspace/proxier.go]: Thanks @danwinship 
Updated!

**Pull request merged**